lib/sacastats_web/views/character_view.ex

```diff
@@ -82,10 +87,13 @@ defmodule SacaStatsWeb.CharacterView do
     """
   end
 
-  defp build_event_log_item(assigns, %Events.Death{} = death, %Session{} = _session) do
+  defp build_event_log_item(assigns, %Death{} = death, %Session{}, character_map) do
+    character_identifier = get_character_name(assigns, character_map, death.character_id)
+    attacker_identifier = get_character_name(assigns, character_map, death.attacker_character_id)
+
     ~H"""
     <li>
-      <%= death.attacker_character_id %> killed <%= death.character_id %>
+      <%= attacker_identifier %> killed <%= character_identifier %>
       with <%= SacaStats.weapons()[death.attacker_weapon_id]["name"] %> (<%= death.attacker_weapon_id %>)
```

Comment:
Add a case for `weapon_id = 0` to avoid logs like

[dndmackey](http://lvh.me:4000/character/dndmackey) killed [Niccom27](http://lvh.me:4000/character/Niccom27) with (0) - 2022-08-29 05:19:09